_docs/getting-started/accounts.md

```diff
@@ -38,6 +38,12 @@ When you log into cloud.gov for the first time, follow the instructions to set t
 * **If you're logged out and forgot your password:** You can [reset your password](https://account.fr.cloud.gov/forgot-password) to log in with a temporary password, then [change your password](https://account.fr.cloud.gov/change-password).
 * **If you're logged out and your password has expired:** You still [reset your password](https://account.fr.cloud.gov/forgot-password) to log in with a temporary password, then [change your password](https://account.fr.cloud.gov/change-password).
 
+#### If you need to change MFA to a new device
+
+If you need to register a new MFA device and still have access to your old device (e.g. you got a new phone and still have your old phone),
```

Comment:
Define what MFA is. So we could change it to say "If you need to register a new MFA (multi-factor authentication) device ..."

Reply:
I didn't define it because I thought it was defined above. I checked and we're actually using TOTP above, so I'm going to change to that